### DISCUSSION TITLE
Import from a terraform mono repo subfolder

### DISCUSSION THREAD

**Discussion by awspunk** [D10S0VSkY-OSS/Stack-Lifecycle-Deployment#77] [Ideas]
Hi @D10S0VSkY-OSS , Do you plan on adding importing modules for stacks from sub folders inside a terraform mono repo ?

**Comment by D10S0VSkY-OSS** [comment]
@awspunk 
WIP

**Comment by D10S0VSkY-OSS** [comment]
Hi @awspunk,
in the latest version v2.8.0 you already have the possibility to use a mono repo by passing to a deploy the path of the folder that contains the terrafom IaC code

[Screencast from 19-07-22 21:23:48.webm](https://user-images.githubusercontent.com/79284025/179833330-a42fc0be-79be-4b92-95c2-75cf801e6902.webm)

Regards,

**Comment by nileshcrptm** [comment]
Thanks @D10S0VSkY-OSS , will check that out
